pkg/controllers/cassandra/nodepool/nodepool.go

```diff
@@ -68,6 +79,61 @@ func (e *defaultCassandraClusterNodepoolControl) removeUnusedStatefulSets(
 	return nil
 }
 
+// Return the lowest int from a slice of ints
+func min(v []int) (m int) {
+	if len(v) > 0 {
+		m = v[0]
+	} else {
+		panic("min called with empty slice")
+	}
+	for _, e := range v {
+		if e < m {
+			m = e
+		}
+	}
+	return
+}
+
+func (e *defaultCassandraClusterNodepoolControl) labelSeedNodes(
+	cluster *v1alpha1.CassandraCluster,
+	set *appsv1beta1.StatefulSet,
+) error {
+	var nodepoolPods []*v1.Pod
+	allPods, err := e.pods.Pods(cluster.Namespace).List(labels.Everything())
+	if err != nil {
+		return err
+	}
+	for _, pod := range allPods {
+		if metav1.IsControlledBy(pod, set) {
+			nodepoolPods = append(nodepoolPods, pod)
+		}
+	}
+
+	// if there are no pods, return early
+	if len(nodepoolPods) < 1 {
+		return nil
+	}
+
+	// Choose the lowest StatefulSet member and mark it as a seed
+	numberedPods := make(map[int]*v1.Pod)
+	ns := []int{}
+	for _, p := range nodepoolPods {
+		elements := strings.Split(p.Name, "-")
+		n, err := strconv.Atoi(elements[len(elements)-1])
+		if err != nil {
+			return err
+		}
+		numberedPods[n] = p
+		ns = append(ns, n)
+	}
+
+	pod := numberedPods[min(ns)]
+	pod.Labels["seed"] = "true"
+	e.kubeClient.CoreV1().Pods(pod.Namespace).Update(pod)
```

Comment:
Elsewhere, we take special care not to mutate the objects returned by informers / listers. 
Doing so will change the object in the shared cache.
Instead look at making a `DeepCopy`.

Reply:
Also need to handle the error returned by the update.

Reply:
And probably should only perform the update if the pod does not already have the label.